### DISCUSSION TITLE
How to lock a product

### DISCUSSION THREAD

**Discussion by stashblox** [saleor/saleor#6102] [Q&A]
We would like to add the ability to lock a product. How I can do that with a plugin ?
Ideally, we need a hook `before_product_updated` to check the status and eventually throw an error if the product is locked.
Without this hook what workaround we could use ?
Thank you in advance for the  help.

**Comment by karolkielecki** [comment]
@stashblox Hi there! What exactly would you like to achieve? Can you describe it in detail?
